client/src/App.js

```diff
@@ -20,7 +20,13 @@ function App() {
 			<Router>
 				<Switch>
 					<Route path="/" exact component={Home} />
-					<Route path="/login" exact component={Login} />
```

Comment:
please be careful while solving the conflicts .please return it instead of the other path